docs/api.md

```diff
@@ -3214,7 +3214,7 @@ Exception is immediately thrown if the request interception is not enabled.
 
 #### request.continue([overrides])
 - `overrides` <[Object]> Optional request overwrites, which can be one of the following:
-  - `url` <[string]> If set, the request url will be changed
+  - `url` <[string]> If set, the request url will be changed in a way that's not observable by page.
```

Comment:
~~by page~~ to the page
I'm not sure what this is saying to me. How could the url have been observable to the page? Is there some part of the fetch API that returns the final url requested?

Reply:
Maybe mention this is not a **redirect**.